client/signup/config/steps-pure.js

```diff
@@ -585,45 +578,6 @@ export function generateSteps( {
 			fulfilledStepCallback: isSiteTopicFulfilled,
 		},
 
-		// Steps with preview
-		// These can be removed once we make the preview the default
-		'site-topic-with-preview': {
-			stepName: 'site-topic-with-preview',
-			providesDependencies: [ 'siteTopic', 'themeSlugWithRepo' ],
-			optionalDependencies: [ 'themeSlugWithRepo' ],
-			fulfilledStepCallback: isSiteTopicFulfilled,
-			props: {
-				showSiteMockups: true,
```

Comment:
This PR removes the last of the steps that define the `showSiteMockups` prop. All remaining references to this term are either just passing the prop along or checking whether it's true. Since it'll always be falsy now we can remove this dead code if we want.

Reply:
Actually, it looks like you're doing that work in #59935. Never mind.